net/lnet/ut/lnet_ut.c

```diff
@@ -265,15 +265,25 @@ do {								\
 } while (0)
 #define zUT(x) zvUT(x, 0)
 
+#ifndef PAGE_SHIFT
+  #ifdef  CONFIG_X86_64
+    #define PAGE_SHIFT    12
+  #else /*aarch64*/
+    #define PAGE_SHIFT    16
+#endif
+  #define PAGE_SIZE	  1UL << PAGE_SHIFT
```

Comment:
We maybe need this:
```
#define PAGE_SIZE (1UL << PAGE_SHIFT)
```
Otherwise, in some special cases, this Macro will yield error.

I mentioned this in last comment for this place.